frontend/src/components/DefenceBox/DefenceMechanism.css

```diff
@@ -30,6 +31,7 @@
 
 .defence-mechanism-fieldset details[open] > summary {
 	border-style: inset;
+	background-color: var(--control-body-background-colour);
```

Comment:
_sneaks tangential styles in without anyone noticing_ 🤭

Reply:
🤫